### DISCUSSION TITLE
Allow to add custom address-bar "textures"

### DISCUSSION THREAD

**Discussion by Skyyblaze** [Maplespe/DWMBlurGlass#262] [Ideas]
Would it be within the scope of the project to allow DWMBlurGlass to handle custom "textures" for the address-bar?

I'm using DWMBlurGlass along with ExplorerBlurMica and while it makes Explorer look amazing with a custom-theme I really would like a distinctive address and search-bar regardless of any custom-theme. This is how my address- and searchbar look right now:

![grafik](https://github.com/Maplespe/DWMBlurGlass/assets/22203339/529d2674-3a84-4bb4-bdcd-1125af6e80a6)

And in addition to the glass-texture feature could DWMBlurGlass also render a texture behind the address- and searchbar like here?

![grafik](https://github.com/Maplespe/DWMBlurGlass/assets/22203339/c7952eff-ebc3-4647-80dc-ba31c3a6037b)

**Comment by Maplespe** [comment]
DWM can only render backgrounds for NC zones, the contents of client zones are handled and drawn by their corresponding user processes (explorer.exe).
So it is out of the scope of this project.

**Reply by Skyyblaze** [reply]
Again I understand, thanks for the clarification!
